generic-styles/content-baked.less

```diff
@@ -190,7 +190,6 @@ span[data-type="media"] img {
 #blockish {
   .style(note) {
     margin: 3rem 0;
-    padding: 1.5rem;
```

Comment:
the top of this is replaced on line 207, the sides on line 206/216, the remaining amounts on those lines are carrying over what was already there, i don't see the bottom ever being replaced in the other changes, and the test is off by 15px, are you sure about your assessment of the margin collapsing being the problem? maybe you're just missing 1.5rem on the bottom from your changes?

Reply:
Production:
![Zrzut ekranu z 2021-09-14 14-42-51](https://user-images.githubusercontent.com/36712200/133260029-d2f182d0-3fd3-4a10-8c06-6efce8ab93a1.png)

After changes:
![Zrzut ekranu z 2021-09-14 14-43-24](https://user-images.githubusercontent.com/36712200/133260085-8e7d6e0e-61b1-41c8-ad0e-35a920f878c6.png)